Customizing/global/plugins/Services/Cron/CronHook/ReportMaster/classes/ReportBase/class.ilObjReportBase.php

```diff
@@ -91,6 +91,14 @@ public function getSettingsData() {
 		return $this->settings;
 	}
 
+	public function getSettingsDataFor($key) {
```

Comment:
@dkloepfer 
Hallo Denis, wir brauchten diese Änderung kurzfristig, deswegen habe ich sie angenommen. Ich würde dich jedoch bitten, dir die neue Methode anzusehen, zu bewerten ob sie in deine Logik passt und ggf. entsprechende Anpassungen vorzunehmen. Danke!